pages/learn/more/collected-data.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Collected Data
+---
+# Data collected by balena
+
+In order to make data driven decisions, balena collects usage related events via a 3rd party analytics service 
+[Mixpanel](https://mixpanel.com/). These events provide us context on the state of application execution steps. 
```

Comment:
I _think_ reference-style links are preferred, though I don't know that's a hard, general requirement

Reply:
@xginn8 would you point me to an example?

Reply:
[like this](https://github.com/balena-io/docs/blame/master/pages/learn/getting-started.md#L67)

Reply:
Thanks, fixed.